Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,996 @@
+                        outputs_key_field="id",
+                        outputs=[],
+                        readable_output=readable_output,
+                    )
+                )
+            break
+
+        readable_output = tableToMarkdown("List of assets", assets, headers=headers, headerTransform=pascalToSpace)
+
+        return_results(
+            CommandResults(
+                outputs_prefix="DSPM.Assets",
+                outputs_key_field="id",
+                outputs=assets,
+                readable_output=readable_output,
+            )
+        )
+        page += 1
+
+
+def dspm_get_data_types_findings_command(client, args):
+    page = 0
+    all_data_type_findings = []
+    headers = ['No', 'Key']
+
+    while True:
+        data_type_findings = get_data_type_findings(
+            client, args, page
+        )
+
+        if not data_type_findings:
+            if page == 0:
+                readable_output = tableToMarkdown("Data Types", [], headers=headers)
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.DataTypesFindings",
+                        outputs_key_field="Key",
+                        outputs=[],
+                        readable_output=readable_output,
+                    )
+                )
+            break
+
+        all_data_type_findings.extend(data_type_findings)
+
+        data_type_findings_formatted = [
+            {
+                "No": index + 1 + (page * MAX_PAGE_SIZE),
+                "Key": dt["dataTypeName"],
+            }
+            for index, dt in enumerate(data_type_findings)
+        ]
+
+        readable_output = tableToMarkdown("Data Types", data_type_findings_formatted, headers=headers)
+        return_results(
+            CommandResults(
+                outputs_prefix="DSPM.DataTypesFindings",
+                outputs_key_field="Key",
+                outputs=data_type_findings_formatted,
+                readable_output=readable_output,
+            )
+        )
+        page += 1
+
+
+def dspm_get_list_of_alerts_command(client, args):
+    page = 0
+    headers = ["id", "detectionTime", "policyName", "assetName", "assetLabels", "cloudProvider", "destinationProjects",
+               "cloudEnvironment", "policySeverity", "policyCategoryType", "status", "eventActor", "eventUserAgent",
+               "eventActionMedium", "eventSource", "policyFrameWorks", "eventRawData"]
+    while True:
+        alerts = get_list_of_alerts(client, args, page)
+        if not alerts:
+            if page == 0:
+                readable_output = tableToMarkdown("List Of Alerts", [], headers=headers, headerTransform=pascalToSpace)
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.Alerts",
+                        outputs_key_field="id",
+                        outputs=[],
+                        readable_output=readable_output,
+                    )
```

Comment:
```suggestion
                    CommandResults(
                        readable_output=readable_output,
                    )
```
in this case we need only the readable_output

Reply:
Sure, we will remove the rest of the parameters.